core/isolate.rs

```diff
@@ -85,6 +85,7 @@ pub trait Behavior {
 /// JavaScript.
 pub struct Isolate<B: Behavior> {
   libdeno_isolate: *const libdeno::isolate,
+  shared_libdeno_isolate: Arc<Mutex<Option<*const libdeno::isolate>>>,
```

Comment:
Should/can the Option be the outermost wrapper?

Reply:
The `Option` is atomically set to `None` when the `Isolate` is dropped. This prevents the IsolateHandle from using a disposed v8 isolate. see [L100-101](https://github.com/denoland/deno/pull/1982/files#diff-7170d073ffb6b76573055a7aaba20d7fR100)

Reply:
Ok sounds good.